python/pyspark/sql/types.py

```diff
@@ -305,7 +305,7 @@ def jsonValue(self):
     @classmethod
     def fromJson(cls, json):
         return ArrayType(_parse_datatype_json_value(json["elementType"]),
-                         json["containsNull"])
+                         json.get("containsNull", True))
```

Comment:
Just to clarify, does this cause any user-facing error? This isn't an API so we're good if it works internally.